2016-11-18 12:00:43,994 - 47 - INFO - log file:/data/BI/vdw/log/buff/buff.dw_user/20161118/signal_table_load_handler.py_20161118120043.log
2016-11-18 12:00:43,994 - 48 - INFO - /data/BI/vdw/bat/signal_table_load_handler.py buff.dw_user
2016-11-18 12:00:43,994 - 49 - INFO - tmp_ld_sql_file:/data/BI/vdw/tmp/buff/buff.dw_user.ld.sql20161118120043
2016-11-18 12:00:43,994 - 50 - INFO - UOW:20161115000000,20161116000000
2016-11-18 12:00:43,994 - 96 - INFO - extract_file:/data/BI/vdw/in/buff/buff.dw_user/20161116000000/buff.dw_user.dat
2016-11-18 12:00:43,995 - 101 - INFO - extract record:24

2016-11-18 12:00:43,995 - 110 - INFO - command: psql -h 172.16.15.131 -p 5432 -d bidw -v ON_ERROR_STOP=1 -U vu_buff -f /data/BI/vdw/tmp/buff/buff.dw_user.ld.sql20161118120043
2016-11-18 12:00:43,995 - 111 - INFO - {'{UOW_FROM}': '20161115000000', '{LEVEL_ID}': 'buff', '{TABLE_ID}': 'dw_user', '{UOW_TO}': '20161116000000', '{ETL_ID}': 'buff.dw_user'}
2016-11-18 12:00:44,000 - 121 - INFO - psql:/data/BI/vdw/tmp/buff/buff.dw_user.ld.sql20161118120043:1: ERROR:  permission denied for relation buff_user
2016-11-18 12:00:44,000 - 126 - ERROR - load executed false.
